csfle/dotnet/aws/reader/CSFLE/InsertEncryptedDocument.cs

```diff
@@ -12,110 +13,88 @@ class InsertEncryptedDocument
 
         public static void Insert()
         {
-            var connectionString = "<Your MongoDB URI>";
+            var credentials = new YourCredentials().GetCredentials();
+            var connectionString = credentials["MONGODB_URI"];
             // start-key-vault
             var keyVaultNamespace = CollectionNamespace.FromFullName("encryption.__keyVault");
             // end-key-vault
             var coll = "patients";
             var db = "medicalRecords";
-            var db_namespace = $"{db}.{coll}";
+            var dbNamespace = $"{db}.{coll}";
 
             // start-kmsproviders
             var kmsProviders = new Dictionary<string, IReadOnlyDictionary<string, object>>();
             var provider = "aws";
+            var awsAccessKey = credentials["AWS_ACCESS_KEY_ID"];
+            var awsSecretAccessKey = credentials["AWS_SECRET_ACCESS_KEY"];
             var awsKmsOptions = new Dictionary<string, object>
             {
-               { "accessKeyId", "<Your AWS Access Key ID>" },
-               { "secretAccessKey", "<Your AWS Secret Access Key>" }
+               { "accessKeyId", awsAccessKey },
+               { "secretAccessKey", awsSecretAccessKey }
             };
             kmsProviders.Add(provider, awsKmsOptions);
             // end-kmsproviders
 
 
             // start-schema
-            var keyId = "<Your base64 DEK ID here>";
-            var schema = new BsonDocument
```

Comment:
As the sample application state is now derived from the testing state rather than the reader state, CSFLE applications use a different schema in docs and sample application. This can be fixed by having CSFLE tests work more similarly to QE tests where the DEK is queried for through an unencrypted client prior to constructing the schema.

Here is a ticket to perform this task: TODO MAKE TICKET